### DISCUSSION TITLE
LogicFlow的浏览器兼容性

### DISCUSSION THREAD

**Discussion by towersxu** [didi/LogicFlow#1143] [General]
目前我们在开发后，只测试了LogicFlow在chrome中的表现效果，不排除在其他浏览器上存在问题。目前我们已知的兼容性信息：

- [在safari浏览器中，HTML节点可能会出现无法拖动的问题](https://github.com/didi/LogicFlow/issues/363)。

- [在火狐浏览中导出图片存在问题](https://github.com/didi/LogicFlow/issues/269)  （已修复）

- 从1.0版本开始，LogicFlow不再兼容IE11，0.6版本是支持的。

- LogicFlow目前不支持移动端的touch事件，所以会出现无法移动画布、节点。

**Comment by Jusluo** [comment]
你好，目前还是不支持手机端的touch事件么，还有其他常见的问题和注意事项么

**Reply by boyongjiong** [reply]
目前还不支持，预计今年会支持移动端。移动端可能浏览器的兼容性也会是个问题，欢迎遇到问题给我们提 issue，这样会方便记录和其他同学预知相关问题。
